### ISSUE TITLE
Training data for image enhancement

### ISSUE THREAD

**Issue by YiruS** [vision-x-nyu/image-sculpting#8]
Hi there,

First of all, thanks so much for providing the high-quality results and detailed codebase! 

There's one question that seems not mention in the paper. What is the training data that is used for DDIM plus process (image enhancement process)? And how many images were used in the fine-tuning? 

Thanks and looking forward to your reply!

**Comment by domejiraphon** [comment]
Hi, we are fine-tuning the model using DreamBooth. The code we used is from [Diffusers implementation](https://github.com/huggingface/diffusers/blob/main/examples/dreambooth/README_sdxl.md) For the instance image, we only use the input image and generate a few hundred images for the prior loss.

**Comment by YiruS** [comment]
Thank you for your response @domejiraphon! 

Another question regarding 3d mesh reconstruction: is the image-sculpting generalizable to mesh generated by other models, instead of zero123? The reason asking is that I tried to use other 3d model such as real3d and CRM but it seems that directly replacing mesh with the original model.obj provided in your codebase, the image quality seems quite not good. Hence wondering if there is anything additional needs to be done?

My understanding is that Dreambooth is used to only maintain the image identity for the initial image, but for coarse mesh to the final inpainting one the credit solely goes to feature injection and this may be the bottleneck for performance. Appreciate your thoughts and insights!

**Comment by domejiraphon** [comment]
Hi, I believe our pipeline can be applied to any single-view reconstruction. All we need is a coarse mesh to guide the subsequent diffusion process. I’ve tried Stable Zero123, and it seemed to work well for me. Other than that, I haven't tested other approaches. Also, no additional steps are required.

Yes, your intuition is correct. Since we unintentionally inject artifacts from the coarse mesh (artifacts from 3d reconstruction and mesh deformation), these artifacts can carry over to the refined image
